lars-reimann/api-editor
# refactor: Qualified and unique path "pname" variable added to most api elements

Closes #449.

### Summary of Changes

Added pname parameter to module, class, function and parameter api elements. pname parameter should be a qualified and unique pathname, unlike qname and name.

pname format: "package/module/class/function/parameter"
module can be: "module.name"

### Testing Instructions

Execute:
parse-package api <package> <out> 

And check that the pname parameter is set for every module, class, function and parameter.
